src/SldStyleParser.ts

```diff
@@ -1001,7 +1001,7 @@ export class SldStyleParser implements StyleParser<string> {
     if (this.withGeoServerVendorOption) {
       const graphicFillPadding = getVendorOptionValue(sldSymbolizer, 'graphic-margin');
       if (!isNil(graphicFillPadding)) {
-        fillSymbolizer.graphicFillPadding = graphicFillPadding.split(',').map(numberExpression);
+        fillSymbolizer.graphicFillPadding = graphicFillPadding.split(/[,\s]/).map(numberExpression);
```

Comment:
Great catch @sebastianfrey! Turns out GeoServer actually expects only spaces and no commas (validating the style does not show any errors, applying it however does). Could you adjust this accordingly?

Reply:
@jansule Adopted the write path for the `graphic-margin` prop. Is this the change you were looking for?